kubetest2-gke/deployer/deployer.go

```diff
@@ -185,8 +190,10 @@ func bindFlags(d *deployer) *pflag.FlagSet {
 	flags.StringVar(&d.network, "network", "default", "Cluster network. Defaults to the default network.")
 	flags.StringVar(&d.environment, "environment", "staging", "Container API endpoint to use, one of 'test', 'staging', 'prod', or a custom https:// URL")
 	flags.StringVar(&d.project, "project", "", "Project to deploy to.")
-	flags.StringVar(&d.region, "region", "", "For use with gcloud commands")
-	flags.StringVar(&d.zone, "zone", "", "For use with gcloud commands")
+	flags.StringVar(&d.region, "region", "", "For use with gcloud commands to specify the cluster region.")
+	flags.StringVar(&d.zone, "zone", "", "For use with gcloud commands to specify the cluster zone.")
+	flags.IntVar(&d.nodes, "node-num", defaultNodePool.Nodes, "For use with gcloud commands to specify the number of nodes for the cluster.")
```

Comment:
let's just name it `num-nodes` for consistency.

Reply:
Done.